cloud/blockstore/config/diagnostics.proto

```diff
@@ -78,6 +78,15 @@ message TMonitoringUrlData
 
     optional string MonitoringYDBProject = 8;
     optional string MonitoringYDBGroupDashboard = 9;
+
+    // monitoring dashboard url template
```

Comment:
The first word should be capitalised. Need an exhaustive list of what can be templated